Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,74 @@
-import React from 'react';
+import React, { useReducer} from 'react';
```

Comment:
good use of hooks. If you understand the reducer pattern then you will do well in redux